src/js/clickable-component.js

```diff
@@ -132,6 +132,18 @@ class ClickableComponent extends Component {
     }
   }
 
+  /**
+   * Handles language change in ClickableComponent for the player in components
+   *
+   *
+   */
+  handleLanguageChange() {
```

Comment:
I think this should use a lowercase `c` ie: `handleLanguagechange`, as I think that we try to name things `handle<uppercase first letter of event>`

Reply:
Looks like we're mostly consistent with that but I see we have a `handleTechTimeUpdate_` method, lol.